src/utils.ts

```diff
@@ -480,21 +480,27 @@ const isLumigoStackTrace = (input) => {
   return LUMIGO_STACK_PATTERNS.some((word) => word.test(input));
 };
 
+export const removeLumigoFromError = (err) => {
```

Comment:
Thinking about this again, I think it's confusing to have a function that takes `err`, modifies it and returns the same object.
I would consider changing it to a pure function that sanitizes the stack as a `string`, and returns a new string that the caller would use to override the original value. WDYT?

Reply:
That makes sense, no need to move around the entire `err` object. Changed, thanks!